scheduler/pkg/envoy/xdscache/resource.go

```diff
@@ -781,28 +791,106 @@ func createTapConfig() *anypb.Any {
 	return tapAny
 }
 
-func createAccessLogConfig() *anypb.Any {
-	accessFilter := accesslog_file.FileAccessLog{
-		Path: "/tmp/envoy-accesslog.txt",
+func createAccessLogConfig(path string) *anypb.Any {
+	config := accesslog_file.FileAccessLog{
+		Path: path,
 		AccessLogFormat: &accesslog_file.FileAccessLog_LogFormat{
 			LogFormat: &core.SubstitutionFormatString{
 				Format: &core.SubstitutionFormatString_TextFormatSource{
 					TextFormatSource: &core.DataSource{
 						Specifier: &core.DataSource_InlineString{
-							InlineString: "[%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\"\n",
+							InlineString: "[%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %GRPC_STATUS_NUMBER% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\"\n",
 						},
 					},
 				},
 			},
 		},
 	}
-	accessAny, err := anypb.New(&accessFilter)
+	accessAny, err := anypb.New(&config)
 	if err != nil {
 		panic(err)
 	}
 	return accessAny
 }
 
+func createAccessLogFilterForErrors() *accesslog.AccessLogFilter {
```

Comment:
Suggest to rename this to `createAccessLogFilterMatchErrors()` (or better).  

I was trying to avoid the ambiguity of whether "filtering" means "excluding" (i.e the current name could be interpreted as "This filters errors and only keeps non-errors".